mava/systems/mat/anakin/mat.py

```diff
@@ -208,28 +190,27 @@ def _loss_fn(
                         )
                         * gae
                     )
-                    loss_actor = -jnp.minimum(loss_actor1, loss_actor2)
-                    loss_actor = loss_actor.mean()
+                    actor_loss = -jnp.minimum(loss_actor1, loss_actor2)
```

Comment:
If we change from `loss_actor` -> `actor_loss` can we please do the same for `loss_actor1` and `loss_actor2`